src/components/Book.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const BooksDisplay = () => (
+const Book = () => (
```

Comment:
- [ ] Great job creating the components as requested by the previous reviewer. :clap:  However, I noticed that you are missing one of the previous reviewer's suggestions. This was to ensure that after creating the `<Book>` component, you also needed to ensure that it takes the book data which can be an array of books as a prop and renders the `title` and the `author` of the book on the UI of your app :100: . Your app can look something like this below once you're done :
- Example Implementation
![image](https://github.com/Edgahkipkemoi/react-bookstore/assets/102699958/48ffc020-88bb-4a1b-8b8c-babc350737c7)

- Your App Currently
![image](https://github.com/Edgahkipkemoi/react-bookstore/assets/102699958/984dba80-90ce-4395-8f17-8b2a0a44c8e8)

Reply:
@NahnahAJ can you elaborate how I will about that.

Reply:
@Edgahkipkemoi Sorry for the late reply but I have sent you a response on slack explaining further with some example implementations as well. Let me know if you have any more questions :wink: